turnwithus/lytics.com
# LYT-94: control video on modal open/close

Adds Wistia video API controls for embedded videos and calls methods within modal interactions:

* if a video is playing in a modal and the modal is dismissed, the video will pause
* if a video had previously been paused and the modal dismissed then reopened, the video will automatically start playing at the same point it was paused
* if neither of those scenarios are met, nothing will happen

**To test:**
* go to either the pattern library homepage examples or the Craft homepage
* launch the play video modal
* play the video
* close the modal
* video should stop playing
* reopen modal
* video should resume playing
* the above should happen without console error or other issue
